### PR TITLE
docs: update docs after release

### PR DESCRIPTION
**Pull request by mwwoda**
Merge this when the next version is released (currently it is v4.3.1).

### CODE REVIEW

File: docs/files.md [hunk 1]
```diff
@@ -260,18 +260,23 @@ A file can be unlocked by calling `FilesManager.UnLock(string id)` with the ID o
...
-Create a Shared Link
+Create or update a Shared Link
 --------------------
 
-A shared link for a file can be generated by calling
+A shared link for a file can be created or updated by calling
 `FilesManager.CreateSharedLinkAsync(string id, BoxSharedLinkRequest sharedLinkRequest, IEnumerable<string> fields = null)`.
 
-<!-- sample put_files_id_shared_link_create -->
+<!-- sample put_files_id shared_link_create -->
```
Comment (arjankowski) [inline docs/files.md:269]: Are you sure that `put_files_id shared_link_create` is right here?

This page https://developer.box.com/reference/put-files-id--add-shared-link/ using this sample label 
`put_files_id add_shared_link`

And this page https://developer.box.com/reference/put-files-id--update-shared-link/ is using
`put_files_id update_shared_link`.

However when we want to add sample code here https://developer.box.com/guides/shared-links/create-or-update/, we should add `put_files_id add_shared_link` as well as in first example.

File: docs/folders.md [hunk 2]
```diff
@@ -116,14 +116,14 @@ folder and all of its contents, set the optional `recursive` parameter to `true`
...
+Create or update a Shared Link
 ---------------------------------
 
-You can create a shared link for a folder by calling
+You can create or update a shared link for a folder by calling
 `FoldersManager.CreateSharedLinkAsync(string id, BoxSharedLinkRequest sharedLinkRequest, IEnumerable<string> fields = null)`
 with the ID of the folder and the shared link parameters.
 
-<!-- sample put_folders_id_shared_link_create -->
+<!-- sample put_folders_id shared_link_create -->
```
Comment (arjankowski) [inline docs/folders.md:126]: The same question as previously, is `put_folders_id shared_link_create` should be here?
Or maybe we should use `'put_folders_id add_shared_link` instead?

File: docs/web-links.md [hunk 3]
```diff
@@ -79,16 +80,18 @@ Copy a Web Link
...
 
-Create a Shared Link
+Create or update a Shared Link
 --------------------
 
-A shared link for a web link can be generated by calling
+A shared link for a web link can be created or updated by calling
 `WebLinksManager.CreateSharedLinkAsync(string id, BoxSharedLinkRequest sharedLinkRequest, IEnumerable<string> fields = null)`.
 
+<!-- sample put_web_links_id add_shared_link -->
```
Comment (arjankowski) [inline docs/web-links.md:94]: Here is OK

File: docs/files.md (hunk 1 above)
Comment (mwwoda) [reply docs/files.md:269]: You may be right. I took the labels from the original Jira ticket, but I see that perhaps we should use `put_files_id add_shared_link` instead (or both). I'll try to clarify this.

Comment (mwwoda) [reply docs/files.md]: Fixed

File: docs/folders.md (hunk 2 above)
Comment (mwwoda) [reply docs/folders.md]: Fixed

**Review by arjankowski — APPROVED** [review]
👍

**Pull request merged**